### PR TITLE
CCM nested cc examples

### PR DESCRIPTION
**Pull request by charanya-j**

### CODE REVIEW

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md [hunk 1]
```diff
@@ -70,7 +70,21 @@ The new Cost Category appears.
 1. Select **New Cost Bucket**.
 2. Enter a name for the cost bucket, such as the name of a department.
 3. Define the rules (data sources) for the cost bucket. You could add multiple conditions to this rule using the AND operator.  
-Add a new row for each rule until the cost bucket correctly includes all the costs incurred for this bucket. These rules are defined using the OR operator.  
+Add a new row for each rule until the cost bucket correctly includes all the costs incurred for this bucket. These rules are defined using the OR operator. You can add another cost category as a rule when you define your cost buckets.
```
Comment (rashminandasahoo) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:73]:
```suggestion
Add a new row for each rule until the cost bucket correctly includes all costs incurred. These rules are defined using the OR operator. You can add another cost category as a rule when defining your cost buckets.
```

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md [hunk 2]
```diff
@@ -70,7 +70,21 @@ The new Cost Category appears.
 1. Select **New Cost Bucket**.
 2. Enter a name for the cost bucket, such as the name of a department.
 3. Define the rules (data sources) for the cost bucket. You could add multiple conditions to this rule using the AND operator.  
-Add a new row for each rule until the cost bucket correctly includes all the costs incurred for this bucket. These rules are defined using the OR operator.  
+Add a new row for each rule until the cost bucket correctly includes all the costs incurred for this bucket. These rules are defined using the OR operator. You can add another cost category as a rule when you define your cost buckets. 
+
+   
+:::important important
+* You cannot include a nested cost category as a rule within another cost category if either of these cost categories contains a shared bucket.
```
Comment (rashminandasahoo) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:77]:
```suggestion
* If either of these cost categories contains a shared bucket, you cannot include a nested cost category as a rule.
```

**Review by rashminandasahoo — COMMENTED** [review]
Minor suggestions.

**Review by rashminandasahoo — APPROVED** [review]

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md [hunk 3]
```diff
@@ -70,7 +70,21 @@ The new Cost Category appears.
...
 2. Enter a name for the cost bucket, such as the name of a department.
 3. Define the rules (data sources) for the cost bucket. You could add multiple conditions to this rule using the AND operator.  
-Add a new row for each rule until the cost bucket correctly includes all the costs incurred for this bucket. These rules are defined using the OR operator.  
+Add a new row for each rule until the cost bucket correctly includes all costs incurred. These rules are defined using the OR operator. You can add another cost category as a rule when defining your cost buckets.
+
+   
+:::important important
+* You cannot include a nested cost category as a rule within another cost category if either of these cost categories contains a shared bucket.
+
+ To illustrate, consider the scenario of creating a cost category, `CC1`, which includes a cost bucket, `CB1`. While defining the cost bucket rules for `CB1`, if you intend to nest a cost category, `CC2` that contains a cost bucket, `CB2`, you must ensure that neither `CB1` nor `CB2` includes a shared cost bucket.
```
Comment (sdsingh07) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:79]: Shared buckets are defined at cost category level not at cost bucket level.

> you must ensure that neither `CC1` nor `CC2` includes a shared cost bucket.

File: docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md [hunk 4]
```diff
@@ -70,7 +70,21 @@ The new Cost Category appears.
...
+* You cannot add the same cost category as a rule in the cost bucket.
+* You cannot create cyclic nested cost categories, where a cost category is nested within each other.
+
+ Consider an example where you have a cost category named `CC1`, which includes a cost bucket called `CB1`. Additionally, you have another cost category named `CC2`, which includes a cost bucket called `CB2`. When defining the cost bucket rules, it is essential to avoid adding `CC2` as a rule in `CC1` and adding `CC1` as a rule in `CC2`.
```
Comment (sdsingh07) [inline docs/cloud-cost-management/3-use-ccm-cost-reporting/2-ccm-cost-categories/1-ccm-cost-categories.md:83]: This example is for one level of nesting.

Another example, CC1 -> CC2 -> CC3 -> CC4 -> CC1

**Pull request merged**